bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -121,6 +122,46 @@ contract("Bridge NFT", async function(accounts) {
       assert.equal(result, "v1");
     });
 
+    it("should call the token creator", async function () {
+      const testTokenCreator = await TestTokenCreator.new({ from: anAccount });
+      const tokenCreator = await this.NFTBridge.getTokenCreator(testTokenCreator.address, defaultTokenId);
+      assert.equal(tokenCreator, anAccount);
+    });
+
+    it("should change the Allow Tokens", async function() {
+      const newAllowTokens = await AllowTokens.new();
+      const receipt = await this.NFTBridge.changeAllowTokens(newAllowTokens.address, { from: bridgeManager });
+      truffleAssert.eventEmitted(receipt, "AllowTokensChanged", (ev) => {
+        return (
+          ev._newAllowTokens == newAllowTokens.address
+        );
+      });
+    });
+
+    it("should set the NFT bridge as upgrade", async function() {
+      const receipt = await this.NFTBridge.setUpgrading(true, { from: bridgeManager });
+      truffleAssert.eventEmitted(receipt, "Upgrading", (ev) => {
+        return (
+          ev._isUpgrading
+        );
+      });
+    });
+
+    it("should change the Side Token Factory", async function() {
+      const newSideTokenFactory = await SideNFTTokenFactory.new();
+      const receipt = await this.NFTBridge.changeSideTokenFactory(newSideTokenFactory.address, { from: bridgeManager });
+      truffleAssert.eventEmitted(receipt, "SideTokenFactoryChanged", (ev) => {
+        return (
+          ev._newSideNFTTokenFactory == newSideTokenFactory.address
+        );
+      });
+    });
+
+    it("should get the received selector", async function() {
+      const receipt = await this.NFTBridge.onERC721Received(bridgeOwner, tokenOwner, defaultTokenId, 0);

```

Comment:
this tests something from `IERC721Receiver`, do we need it?
besides, we only check that the transaction was successful, nothing more

Reply:
Well if we want to implement the interface `IERC721Receiver` we need to implement the method `onERC721Received`, and the test just check if it's not broken :/